frontend/src/lib/components/Users/UserModal.svelte

```diff
@@ -27,6 +32,14 @@
     user = _user;
     await userDetailsModal.openModal(true, true);
   }
+
+  const { notifySuccess } = useNotifications();
+
+  // eslint-disable-next-line func-style
```

Comment:
eslint doesn't think so. As far as I can tell, because there's a `function foo() { }` style definition elsewhere in the component, it thinks every function definition needs to be in that style and rejects `bar = (params) => { doSomething(); }`. But since Svelte wants an `export let` here, I can't switch to the `function bar() { }` style here as the Svelte compiler treats that differently.

Basically eslint is fighting with the Svelte compiler, and this was the best way to get eslint to shut up.

Reply:
Oh wait, that was the previous version of the code where I exposed a function prop. Never mind, that comment can go now.

Reply:
Removed in commit 41391908.